ruff.toml

```diff
@@ -81,6 +81,9 @@ ignore = [
     "PLR2004",  # Magic constants
     "PLR6301",  # Method * could be a function, class method, or static method
   ]
+"pytket/phir/cli.py" = [
```

Comment:
For single file ignores, I tend to prefer adding the ignore at the top of the file, like so:
```
# ruff: noqa: T201
```